include/picongpu/param/particle.param

```diff
@@ -230,7 +219,8 @@ namespace picongpu
                  */
                 static constexpr uint32_t numParticlesPerCell = TYPICAL_PARTICLES_PER_CELL;
 
-                const InCellOffset_t inCellOffset;
+                //! each x, y, z in-cell position component in range [0.0, 1.0)
+                static constexpr auto inCellOffset = float3_X(0., 0., 0.);
```

Comment:
should be `static constexpr auto inCellOffset = float3_X(0., 0., 0.).shrink<simDim>();` to be compatible with 2D setups.